tools/mount_gcsfuse/main.go

```diff
@@ -123,8 +127,17 @@ func makeGcsfuseArgs(
 			"custom_endpoint",
 			"config_file",
 			"experimental_metadata_prefetch_on_mount",
-			"kernel_list_cache_ttl_secs":
-			args = append(args, "--"+strings.Replace(name, "_", "-", -1), value)
+			"kernel_list_cache_ttl_secs",
+			"stat_cache_max_size_mb",
+			"type_cache_max_size_mb",
+			"metadata_cache_ttl_secs",
+			"log_severity",
+			"log_rotate_max_file_size_mb",
+			"log_rotate_backup_file_count",
+			"file_cache_max_size_mb",
```

Comment:
do we support flags like `file_cache_max_size`, `type-cache-max-size-mb` ?

Reply:
The flags are behind an environment variable currently, will be supported soon. This change won't make any difference for the end user as they will see an error in both cases (with and without this change). I am adding the flags here as it will make manual testing with persistent mounting easier for us during development.